### PR TITLE
ENH: recover contour in Quick2D (resolves #815)

### LINKED ISSUE #815: Recover contour for Quick2D
https://github.com/wright-group/WrightTools/blob/86ebafa5910fb7614552feec13279f53299e13b6/WrightTools/artists/_quick.py#L312

### PR DESCRIPTION
**Pull request by darienmorrow**

### CODE REVIEW

File: WrightTools/artists/_quick.py [hunk 1]
```diff
@@ -309,7 +309,23 @@ def quick2D(
...
+                else:
+                    limit = data_channel.mag()
+                contour_levels = np.linspace(
+                    -limit + channel.null - 1e-10, limit + channel.null + 1e-10, contours + 2
+                )
+            else:
+                if contours_local:
+                    limit = np.nanmax(zi)
+                else:
+                    limit = data_channel.mag()
```
Comment (ksunden) [inline WrightTools/artists/_quick.py]: Should this be max?

File: WrightTools/artists/_quick.py [hunk 2]
```diff
@@ -309,7 +309,23 @@ def quick2D(
...
+                if contours_local:
+                    limit = channel.mag()
+                else:
+                    limit = data_channel.mag()
+                contour_levels = np.linspace(
+                    -limit + channel.null - 1e-10, limit + channel.null + 1e-10, contours + 2
+                )
+            else:
+                if contours_local:
+                    limit = np.nanmax(zi)
```
Comment (ksunden) [inline WrightTools/artists/_quick.py]: should this be channel.max()?

File: WrightTools/artists/_quick.py [hunk 3]
```diff
@@ -309,7 +309,23 @@ def quick2D(
...
+                else:
+                    limit = data_channel.mag()
+                contour_levels = np.linspace(channel.null - 1e-10, limit + 1e-10, contours + 2)
```
Comment (ksunden) [inline WrightTools/artists/_quick.py]: Why are we avoiding a contour at channel.null?

I get avoiding the maximum and the min/max in signed, but why null?

Comment (darienmorrow) [reply WrightTools/artists/_quick.py]: Were we not to do this, if there is data at (or below) null, then contour would be present and unhelpful.

File: WrightTools/artists/_quick.py (hunk 2 above)
Comment (darienmorrow) [reply WrightTools/artists/_quick.py]: Sure.

File: WrightTools/artists/_quick.py (hunk 1 above)
Comment (darienmorrow) [reply WrightTools/artists/_quick.py]: I think so. I'll change it.

File: WrightTools/artists/_quick.py (hunk 3 above)
Comment (darienmorrow) [reply WrightTools/artists/_quick.py]: TBH, this is how it was written in `mpl_2D` and I just kept the idea.

Comment (ksunden) [reply WrightTools/artists/_quick.py]: if there's data below null here, it will be present anyway

**Pull request merged**